src/pages/patientView/PatientViewPage.tsx

```diff
@@ -43,6 +45,7 @@ interface IPatientViewState {
     mutationData: any;
     mrnaExprRankData?: MrnaRankData;
     activeTabKey: number;
+    hotspotsData: any;
```

Comment:
@onursumer in my mind the jury is still out on whether that's preferable to indexing into a columnProps property. I kind of like columnProps because then we don't have to change the entire data (and thus, I think, force regenerate all the rows) when new data comes in.